wordfreq/chinese.py

```diff
@@ -0,0 +1,18 @@
+from pkg_resources import resource_filename
+import jieba
+import msgpack
+import gzip
+
+DICT_FILENAME = resource_filename('wordfreq', 'data/jieba_zh.txt')
+SIMP_MAP_FILENAME = resource_filename('wordfreq', 'data/_chinese_mapping.msgpack.gz')
+SIMPLIFIED_MAP = msgpack.load(gzip.open(SIMP_MAP_FILENAME), encoding='utf-8')
+JIEBA_TOKENIZER = jieba.Tokenizer(dictionary=DICT_FILENAME)
```

Comment:
Oh, right. That would be a bootstrapping problem. I had forgotten that `wordfreq_builder` imports the file even though it avoids Chinese tokenization.